### ISSUE TITLE
Smart Plug RC ?

### ISSUE THREAD

**Issue by DJJL** [roleoroleo/4-noks_ha_integration#8]
Hi,
thank you for this module which allows me to recover the values ​​of the E4U.
The refresh seems a little slow, but maybe that's normal?
Another question, I have a Smart Plug RC connected to the E4U via the Smart Pro. I can't find the consumption information in the list.
Is it possible to recover it?
It is visible in the 4-Noks application
Thank you so much

**Comment by roleoroleo** [comment]
I'm sorry but I no longer have the device.
If you are able to sniff the network messages at the ethernet port, maybe I can try...

**Comment by DJJL** [comment]
Ok, which screen send you ?
I see that, it's right ?
![image](https://github.com/roleoroleo/4-noks_ha_integration/assets/47599201/5cd939d0-5d10-47eb-ab35-006395f0a3aa)
after 1 hour
![image](https://github.com/roleoroleo/4-noks_ha_integration/assets/47599201/52e2dc05-73b2-48e4-91c9-fb28d11b5fc6)

**Comment by roleoroleo** [comment]
If this device works with SNMP, I can't add the support in this integration.

I don't know what's Smart Pro. Is it a serial connection? Proprietary bus? Or it use an ehternet connection..
Try again. Start the capture and use the app to request the data.
Then, send me the pcap file filtered with the ip of the device.

**Comment by DJJL** [comment]
ah ok, thank you

here is the link to the Smart Pro documentation : 
https://www.4-noks.com/download/fogli-istruzioni-4-noks/fotovoltaico/+05N002010R3.0_Pro-Smart_Instruction-manual_IT-EN-FR-DE_20180709.pdf

there is no application on pc for E4U, so I used Putty so that Wireshark sees the data received

![image](https://github.com/roleoroleo/4-noks_ha_integration/assets/47599201/e6863e21-ea0c-4037-84df-206f09f71152)

**Comment by roleoroleo** [comment]
Can you see Smart Plug RC values in the list above?
Or the list contains just E4U variables?
Maybe there is another command to show these values.
You should use an old hub or a switch with monitor port to capture the traffic on your E4U.
I know, it's not easy...

**Comment by DJJL** [comment]
yes, I just checked, when the Smart pro is OFF or ON, there is a change on the DEVHAO line
DEVHA0;1;0;15;1;0;0;875526;-58;81;;;
to
DEVHA0;1;0;15;1;1;1;875526;-58;81;;; 
and...

![image](https://github.com/roleoroleo/4-noks_ha_integration/assets/47599201/12949443-6231-469c-8c91-d5f0b42291a6)

![image](https://github.com/roleoroleo/4-noks_ha_integration/assets/47599201/27e4d2ed-a46e-4062-94da-28d74db85788)

I see also a difference about the numbers : 
;DEVHA0;1;1;15;1;0;0;877874;-55;81;;;
875526 (before total "Energy" ?) > 87874 (new total "Energy") = 2 348 ("Wh" I suppose ?) it's difference of the "Energy" on 5 days)

another, it's strange for me : 
-55;88;;; > -55;81;;;
